### PR TITLE
Added location in contact

### LINKED ISSUE #85: Add my location to contact section
**Is your feature request related to a problem? Please describe.**
Currently, there is no location mentioned on contact section

**Describe the solution you'd like**
Add a location icon and put like this "Location : Kandy, Sri Lanka"

**Are you working on it?**
No

### PR DESCRIPTION
**Pull request by muskaan04**
## Purpose

The purpose of this Pull Request is to fix #85 

## Goals
Added the location in contact using text in contact setion

## Approach
By text and styling the contact section

  
### Preview Link
https://pr-102-NEW-YOSHITHA-WEBSITE.surge.sh

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: dark/index.html [hunk 1]
```diff
@@ -8,8 +8,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
     <meta
       name="description"
-      content="Hi, I am Yoshitha Rathnayake. I am from Kandy, Sri Lanka. I am 14 years old. I am studing in St. Anthony's College, Kandy"
+      content="Location:- Sri Lanka"/>
```
Comment (YoshithaRathnayake) [inline dark/index.html:11]: Remove this unwanted changes. Keep it like what early there

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: light/index.html [hunk 2]
```diff
@@ -53,6 +53,8 @@
     <!-- Title -->
 
     <title>YOSHITHA RATHNAYAKE</title>
+    <title>Sri Lanka</title>
```
Comment (YoshithaRathnayake) [inline light/index.html:56]: Remove these unwanted changes. Keep it like what early there

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: index.html [hunk 3]
```diff
@@ -9,7 +9,7 @@
     <meta http-equiv="Refresh" content="13; url=dark" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <meta name="description" content="Hi, I am Yoshitha Rathnayake. I am from Kandy, Sri Lanka. I am 13 years old. I am studing in St. Anthony's College, Kandy">
+    <meta name="description" content="Hi, I am Yoshitha Rathnayake. I am from Kandy, Sri Lanka. I am 14 years old. I am studing in St. Anthony's College, Kandy">
```
Comment (YoshithaRathnayake) [inline index.html:12]: Remove these early changes

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Review by muskaan04 — COMMENTED** [review]
changes has been done

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
image
 
Not aligned correcly

**Comment by YoshithaRathnayake** [comment]
This issue is not assigned to you @muskaan04

**Pull request closed without merging**